### PR TITLE
People without access can no longer use R&D consoles research.

### PR DESCRIPTION
**Pull request by Bibby0110**
# Github documenting your Pull Request

It's powergaming

# Wiki Documentation

# Changelog

:cl:  
tweak: You now need science access to research stuff on the R&D console
/:cl:

### CODE REVIEW

**Comment by Jarod1200** [comment]
Does this block engineers from printing circuits from their rnd console and circuit imprinter

**Comment by Bibby0110** [comment]
> Does this block engineers from printing circuits from their rnd console and circuit imprinter

Yes.

**Comment by Bibby0110** [comment]
> > Does this block engineers from printing circuits from their rnd console and circuit imprinter
> 
> Yes.

Not anymore

**Comment by maxion12345** [comment]
isnt there a setting on the rnd board to disable researching, I know engineering's has to be hacked first

maybe if we put it behind some sort of secured area so people that are scientists can only access it

**Comment by Hopekz** [comment]
I know you made this as a funny PR response to #12214, but I think this is unironically a good merge.

Even when I was staff we'd have people noted when caught for this so logically with how the rules are structured it shouldn't really even be an option in the first place.

I don't think this is a big deal but now that it was brought up I don't really see an issue with bringing the code closer to how rules are enforced on the server.

**Comment by boba247** [comment]
Does this effect antags stealing research nodes?

**Comment by githubuser4141** [comment]
Does this affect Free Miners, Free Golems, Charlie Station roles, and every other role that might have RnD access (but isn't a miner tiding into RnD?) Perhaps add a way to hack it so assistants can still get maint/white ship RnD stuff.

**Comment by Bibby0110** [comment]
> Does this affect Free Miners, Free Golems, Charlie Station roles, and every other role that might have RnD access (but isn't a miner tiding into RnD?) Perhaps add a way to hack it so assistants can still get maint/white ship RnD stuff.

Last I checked those all hook into the Stations RnD network, do they not?

**Comment by Cartlord** [comment]
They do all hook into station R&D, as far as I remember.

**Comment by Bibby0110** [comment]
> They do all hook into station R&D, as far as I remember.

Then their consoles should be access restricted.

**Comment by githubuser4141** [comment]
> > They do all hook into station R&D, as far as I remember.
> 
> Then their consoles should be access restricted.

So you'll give them RnD accesses? RnD is a pretty major factor for those roles.

**Comment by Bibby0110** [comment]
> > > They do all hook into station R&D, as far as I remember.
> > 
> > 
> > Then their consoles should be access restricted.
> 
> So you'll give them RnD accesses? RnD is a pretty major factor for those roles.

No

**Comment by Hopekz** [comment]
> So you'll give them RnD accesses? RnD is a pretty major factor for those roles.

Why would they have access to use the station's research points they're not even a part of the station.

They already get the research that the station makes.

File: code/modules/research/rdconsole.dm [hunk 1]
```diff
@@ -1140,8 +1143,6 @@ Nothing else in the console has ID requirements.
 
 /obj/machinery/computer/rdconsole/robotics
 	name = "Robotics R&D Console"
-	req_access = null
-	req_access_txt = "29"
```
Comment (adamsong) [inline code/modules/research/rdconsole.dm:1144]: Why are you removing robo access from the robo's console? Do you not want them researching?

Comment (Bibby0110) [reply code/modules/research/rdconsole.dm:1144]: Yes

Comment (JamieD1) [reply code/modules/research/rdconsole.dm:1144]: Revert this.

**Comment by githubuser4141** [comment]
> > So you'll give them RnD accesses? RnD is a pretty major factor for those roles.
> 
> Why would they have access to use the station's research points they're not even a part of the station.
> 
> They already get the research that the station makes.

So the Free Golems/Miners can't use their own RnD console kit? The one that's uniquely available to them as a reward for getting ores? What if the main station doesn't do any research, this means that these roles suffer because of it.

File: code/modules/research/rdconsole.dm (hunk 1 above)
Comment (Bibby0110) [reply code/modules/research/rdconsole.dm:1144]: No

**Comment by JamieD1** [comment]
Considering you are refusing changes I am going to close this considering robotics and others losing research is a deal breaker.

**Comment by Hopekz** [comment]
> Considering you are refusing changes I am going to close this considering robotics and others losing research is a deal breaker.

You should say that you'd like for a change to happen to the PR before you close it instead of just assuming people will take a maintainer review on game/PR direction. 
The function of a maintainer has always been to review code vs demanding game direction.

**Pull request closed without merging**